From 9b28f18bb2a0a54c05d4a71ff5db7a1bf17cdadd Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Mon, 21 Aug 2017 18:02:04 +0800
Subject: [PATCH 2430/5242] MLK-16242-1 ARM64: dts: imx8mq: add the 8th irq
 for DCSS

commit  29891a101a24a48c4e9be66daecaecee722a3f75 from
https://source.codeaurora.org/external/imx/linux-imx.git

The 8th interrupt of DCSS comes from DTG module
and it is used as a general purpose timings
interrupt which can generate an interrupt at a
programmable X/Y corordinate. There are total
four such interrupts in DTG. And this one will
be used for a VSYNC Active interrupt.

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index fdbdf7a..c38225d 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -413,6 +413,7 @@
 			     <5 IRQ_TYPE_LEVEL_HIGH>,	/* dpr3 */
 			     <6 IRQ_TYPE_LEVEL_HIGH>,	/* ctx_ld */
 			     <7 IRQ_TYPE_LEVEL_HIGH>,	/* rd_src */
+			     <8 IRQ_TYPE_LEVEL_HIGH>,	/* dtg_programmable_1: for vsync */
 			     <15 IRQ_TYPE_LEVEL_HIGH>,	/* dec400d_1 */
 			     <16 IRQ_TYPE_LEVEL_HIGH>,	/* dtrc_2 */
 			     <17 IRQ_TYPE_LEVEL_HIGH>,	/* dtrc_3 */
-- 
1.7.9.5

